{
    "type": "bundle",
    "id": "bundle--af0d07fc-c95f-46e2-99f0-d6939223ad6c",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--cf4caf23-1deb-4e07-b089-30aa1a543ccb",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2021-03-13T20:13:37.268698Z",
            "modified": "2021-03-13T20:13:37.268698Z",
            "name": "CVE-2016-4445",
            "description": "The fix_lookup_id function in sealert in setroubleshoot before 3.2.23 allows local users to execute arbitrary commands as root by triggering an SELinux denial with a crafted file name, related to executing external commands with the commands.getstatusoutput function.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2016-4445"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}